src/app/app.resources.ts

```diff
@@ -1,31 +0,0 @@
-// Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
```

Comment:
We dont need it?

Reply:
This whole file was deleted. I only copied it over because search depends on it, but I think we'll have to re-implement our own search module.